<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoachInterface</class>
 <widget class="QWidget" name="CoachInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>181</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>165</width>
      <height>139</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="applicationsOfLessonsBut">
       <property name="text">
        <string>Заявки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="futureLessonsBut">
       <property name="text">
        <string>Предстоящие занятия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="historyOfLessonsBut">
       <property name="text">
        <string>История занятий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moneyBut">
       <property name="text">
        <string>Заработок</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="applicationsOfLessons">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>40</y>
       <width>611</width>
       <height>471</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>-1</y>
        <width>571</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="coachApplicationsView"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>240</y>
       <width>84</width>
       <height>67</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="acceptApplicationBut">
        <property name="text">
         <string>Принять</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rejectApplicationBut">
        <property name="text">
         <string>Отклонить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="futureLessons">
    <widget class="QTableView" name="coachLessonView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>569</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>87</width>
       <height>67</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="endLessonBut">
        <property name="text">
         <string>Завершить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelLessonBut">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="historyOfLessons">
    <widget class="QTableView" name="coachHistoryLessons">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>569</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="money">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>521</width>
       <height>198</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Выберите период</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Начало</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="coachStartDate">
              <property name="dateTime">
               <datetime>
                <hour>3</hour>
                <minute>0</minute>
                <second>1</second>
                <year>2024</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="time">
               <time>
                <hour>3</hour>
                <minute>0</minute>
                <second>1</second>
               </time>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Конец</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="coachEndDate">
              <property name="dateTime">
               <datetime>
                <hour>3</hour>
                <minute>0</minute>
                <second>1</second>
                <year>2025</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="time">
               <time>
                <hour>3</hour>
                <minute>0</minute>
                <second>1</second>
               </time>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="coachDateBut">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="displayCoachMoney">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
